pkg/aws/ec2/ec2.go

```diff
@@ -494,15 +481,19 @@ func createAWSTagSlice(tags map[string]string) []ec2_types.Tag {
 }
 
 func (c *Client) describeSecurityGroups(ctx context.Context) ([]ec2_types.SecurityGroup, error) {
-	c.limiter.Limit(ctx, "DescribeSecurityGroups")
-	sinceStart := spanstat.Start()
-	output, err := c.ec2Client.DescribeSecurityGroups(ctx, &ec2.DescribeSecurityGroupsInput{})
-	c.metricsAPI.ObserveAPICall("DescribeSecurityGroups", deriveStatus(err), sinceStart.Seconds())
-	if err != nil {
-		return []ec2_types.SecurityGroup{}, err
+	result := []ec2_types.SecurityGroup{}
```

Comment:
same here for the initialization of the slice.

Reply:
Updated